qemu/target-mips/cpu.c

```diff
@@ -151,15 +151,17 @@ void mips_cpu_register_types(void *opaque)
         
         sizeof(MIPSCPUClass),
         sizeof(MIPSCPU),
-		opaque,
+        opaque,
         
-		mips_cpu_initfn,
-		NULL,
-		NULL,
+        mips_cpu_initfn,
+        NULL,
+        NULL,
+
+        NULL,
```

Comment:
yes, but that is OK to be slower here, as mips_cpu_register_types() is only executed once, at init time (i.e uc_open()).